R/render.R

```diff
@@ -86,7 +86,9 @@ render <- function(path = ".", site_url = NULL, type = c("book", "website")) {
   fs::dir_copy(path, temporary_directory)
   withr::with_dir(file.path(temporary_directory, fs::path_file(path)), {
     fs::file_delete(fs::dir_ls(regexp = "\\...\\.qmd"))
-    quarto::quarto_render(as_job = FALSE)
+    metadata <- list("true")
+    names(metadata) <- sprintf("lang-%s", main_language)
```

Comment:
In Pandoc templates we can't have something like "if lang==something" hence my creating such a variable.